src/NzbDrone.Api/Queue/QueueResource.cs

```diff
@@ -17,7 +19,10 @@ public class QueueResource : RestResource
         public decimal Size { get; set; }
         public string Title { get; set; }
         public decimal Sizeleft { get; set; }
+
+        [JsonConverter(typeof(TimeSpanConverter))]
         public TimeSpan? Timeleft { get; set; }
```

Comment:
I thought we could register convertors to a specific type that way we wouldn't have to annotate every usage.